dist/init/systemd/rkt-gc.service

```diff
@@ -1,3 +1,7 @@
+[Unit]
+Description=Garbage Collection for rkt
+
 [Service]
+Environment=GRACE_PERIOD=1d
```

Comment:
Uh oh. I don't think those should have been overloaded. In the case of containers, it's nice to have them around for debugging purposes. Networking cleanup, as you mentioned, should be cleaned up relatively frequently.